app/views/steps/details/confirmation/show.html.erb

```diff
@@ -19,6 +22,8 @@
   </strong>
 </div>
 
+<p class="govuk-body"><%= t '.taxpayer_copy_message'%></p>
```

Comment:
Just nitpicking here but you should fix the spacing ...ge'%> should be ge' %>. Same for the above too.